release.yaml

```diff
@@ -1,13 +1,16 @@
 rancher-monitoring:
-  - 103.0.0+up40.1.2
+- 103.0.0+up40.1.2
 rancher-monitoring-crd:
-  - 103.0.0+up40.1.2
+- 103.0.0+up40.1.2
 fleet:
-  - 102.1.1+up0.7.1
+- 102.1.1+up0.7.1
```

Comment:
Can we remove 102.1.1+up0.7.1 ? since we are only adding 103.0.0+up0.8.0-rc.9 ? and same for fleet-agent ?